test/multiverse/suites/rails/Envfile

```diff
@@ -1,18 +1,28 @@
+# TODO: Determine the root cause of our compatibility issues with rack-test 2+.
```

Comment:
Perhaps this TODO could use the same label as the one mentioned above? I could also see these making it easier for us to pull out potential technical debt issues when writing MMFs.

Reply:
I went with `UNIT TESTS` in 8ad6c42 here as well

Reply:
correction: 8a4aaba for the TODO comment